sdk/src/rest/commands/update/items.ts

```diff
@@ -12,31 +12,41 @@ export type UpdateItemOutput<
  * Update multiple items at the same time.
  *
  * @param collection The collection of the items
- * @param keys The primary key of the items
+ * @param keysOrQuery The primary key of the items
```

Comment:
Description needs to be updated
```suggestion
 * @param keysOrQuery The primary keys or a query
```